### PR TITLE
test: enable k8s for clh

### PR DESCRIPTION
**Pull request by jcvenegas**
Enable k8s for clh.

Fixes: #2318

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

**Comment by GabyCT** [comment]
@jcvenegas fixes 1?

File: .ci/jenkins_job_build.sh [hunk 1]
```diff
@@ -152,6 +152,17 @@ case "${CI_JOB}" in
...
 	export DEFSANDBOXCGROUPONLY=true
 	;;
+"CLOUD-HYPERVISOR")
+	export CRIO="no"
+	export CRI_CONTAINERD="yes"
+	export CRI_RUNTIME="containerd"
+	export KATA_HYPERVISOR="cloud-hypervisor"
+	export KUBERNETES="yes"
+	export OPENSHIFT="no"
+	export TEST_CRIO="false"
```
Comment (chavafg) [inline .ci/jenkins_job_build.sh]: this is by default false in the crio test script, so should be fine if we remove it

File: .ci/jenkins_job_build.sh [hunk 2]
```diff
@@ -152,6 +152,17 @@ case "${CI_JOB}" in
...
+	export experimental_kernel="true"
+	export use_runtime_class="true"
```
Comment (chavafg) [inline .ci/jenkins_job_build.sh]: we now always use `runtimeClass`, so this flag is no longer needed.

File: .ci/jenkins_job_build.sh (hunk 1 above)
Comment (jcvenegas) [reply .ci/jenkins_job_build.sh]: I prefer to keep it explicit it helps know what is actually being tested for the CI_JOB

Comment (chavafg) [reply .ci/jenkins_job_build.sh]: ok, agree 👍

**Pull request merged**